pkg/arvo/sys/arvo.hoon

```diff
@@ -1555,9 +1555,9 @@
         ^-  roon
         |=  [lyc=gang pov=path car=term bem=beam]
         ^-  (unit (unit cage))
-        ?.  ?|  =(our p.bem)
+        ?.  ?&  =(our p.bem)
                 ?=(%$ q.bem)
-                =([%da now] p.r.bem)
+                =(now p.r.bem)
```

Comment:
This needs to check tag too, otherwise we'd be incorrectly responding to some possible `%ud` and `%uv` cares.